Examples/Run/Show/CMakeLists.txt

```diff
@@ -20,4 +20,4 @@ install(
   TARGETS
     ActsExampleShowFatrasGeneric
     ActsExampleShowParticles
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
I guess this change could go?

Reply:
yes, I removed the newline in the file, added it back.